modules/common/istio/main.tf

```diff
@@ -23,6 +41,56 @@ resource "helm_release" "istio" {
     value = "${var.crd_waiter}"
   }
 
+  set {
+    name  = "gateways.istio-egressgateway.enabled"
+    value = "false"
+  }
+
+  set {
+    name  = "gateways.istio-ingressgateway.sds.enabled"
+    value = "true"
+  }
+
+  set {
+    name  = "global.k8sIngress.enabled"
+    value = "true"
+  }
+
+  set {
+    name  = "global.k8sIngress.enableHttps"
+    value = "true"
+  }
+
+  set {
+    name  = "global.k8sIngress.gatewayName=ingressgateway"
+    value = "true"
```

Comment:
I believe this gatewayName is being set wrong, the name is supposed to be "global.k8sIngress.gatewayName" and the value itself is supposed to be "ingressgateway" or our ingress gateway name. Also not sure if these values need to be set up atm since we didn't have the cert manager fully working. Referenced from here -  https://istio.io/docs/tasks/traffic-management/ingress/ingress-certmgr/